src/python/pants/backend/jvm/targets/junit_tests.py

```diff
@@ -65,12 +71,21 @@ def __init__(self, cwd=None, test_platform=None, payload=None, timeout=None,
       if value is not None:
         extra_env_vars[key] = str(value)
 
+    deprecated_conditional(lambda: 'test_platform' in kwargs, '1.27.0.dev0', 'test_platform',
```

Comment:
Master just bumped to `1.26.x`, so should target `1.28.0.dev0` here I think.

Reply:
k